doc/src/dump.rst

```diff
@@ -363,22 +363,25 @@ attributes is given below.
 
 .. versionadded:: 22Dec2022
 
-For style *grid* the extent of the Nx by Ny by Nz grid that overlays
-the simulation domain is output with each snapshot:
+For style *grid* the dimension and extent of the Nx by Ny by Nz grid
```

Comment:
I think the terms "dimension" and "extent" need additional qualifiers here or alternate terms to avoid confusion.
People often use the term "grid dimensions" to describe what you have called "extent" here and renamed to "size" in the dump output. After all "ITEM: DIMENSION" does not seem to refer to the grid but to the dimensionality of the simulation cell.